centreon/packages/ui/src/utils/calculateDaysUntilLicenseExpiration.ts

```diff
@@ -0,0 +1,25 @@
+import { path, pipe, propEq, find } from 'ramda';
+
+interface Props {
+  module: string;
+  response;
+}
+export const calculateDaysUntilLicenseExpiration = ({
+  response,
+  module
+}: Props): number => {
+  const getExpirationDate = pipe(
+    path(['result', 'module', 'entities']),
+    find(propEq('id', module)),
+    path(['license', 'expiration_date'])
+  ) as (data) => string;
+
+  const currentDate = new Date();
```

Comment:
Use dayjs for date management

Reply:
Should we add a unit test for this util or do we handle the coverage at the component level?

Reply:
my current implementation involves an initial test of the function on a standalone, generic component (an empty div), devoid of any additional behaviors.  and then proceed to test it again on every page within each module.